core/src/validator.rs

```diff
@@ -108,6 +109,7 @@ pub struct ValidatorConfig {
     pub bpf_jit: bool,
     pub send_transaction_retry_ms: u64,
     pub send_transaction_leader_forward_count: u64,
+    pub supported_indexes: Vec<IndexType>,
```

Comment:
Another example of where `supported_indexes` is a little too generic:
```suggestion
    pub account_indexes: Vec<accounts_index::IndexType>,
```

Reply:
Done in https://github.com/solana-labs/solana/pull/14212/commits/d204a08432c3936cd1450cf16e1a67236e758461